packages/examples/packages/webpack-plugin/webpack.config.ts

```diff
@@ -35,7 +35,7 @@ const config: Configuration = {
     minimize: true,
     minimizer: [
       new TerserPlugin({
-        minify: TerserPlugin.swcMinify,
+        parallel: true,
```

Comment:
Why did you change this?

Reply:
Same reason we can't use the SWC minifier in the CLI. It screws with our post processing of HTML comments